src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -125,7 +125,10 @@ class RawTableRelation(
     val (minLastUpdatedTime, maxLastUpdatedTime) = filtersToTimestampLimits(filters, "lastUpdatedTime")
 
     val rdd =
-      readRows(config.limitPerPartition, None, RawRowFilter(minLastUpdatedTime, maxLastUpdatedTime))
+      readRows(
+        config.limitPerPartition,
+        None,
+        RawRowFilter(minLastUpdatedTime, maxLastUpdatedTime, Some(requiredColumns)))
```

Comment:
You can exclude "key" and "lastUpdatedTime".

We should also request all columns if the `requiredColumns` array is empty, since rely on it on line 122